### PR TITLE
Fix $supported array spelling in sample

### PR DESCRIPTION
**Pull request by R-J**
The sample had shown `$Supported` as spelling for the `$supported` array.

### CODE REVIEW

File: packages/modxdiscuss.php [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+/**
+ * MODX Discuss exporter tool.
+ *
+ * @copyright 2017 Vanilla Forums Inc.
+ * @license http://opensource.org/licenses/gpl-2.0.php GNU GPL2
+ * @package VanillaPorter
+ */
+
+// Add to the $Supported array so it appears in the dropdown menu. Uncomment next line.
+$Supported['modxdiscuss'] = array('name'=> 'MODX Discuss Extension', 'prefix'=>'modx_discuss_');
```
Comment (topcheese13) [inline packages/modxdiscuss.php]: $Supported should start with a lower case.

File: packages/modxdiscuss.php [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+
+// Optionally, add the features you are supporting. Set all values to 1 or a string for support notes.
+// See functions/feature-functions.php VanillaFeatureSet() for array keys.
+$Supported['modxdiscuss']['features'] = array(
```
Comment (topcheese13) [inline packages/modxdiscuss.php]: $Supported should start with a lower case.

**Comment by topcheese13** [comment]
Hi @R-J, I left 2 inline comments regarding casing.

Other than than I would have 1 more pointer.

For example, when doing:

```
$comment_Map = array(
            'Thread_ID' => 'CommentID',
            'Topic_ID' => 'DiscussionID',
            'Author_ID' => 'InsertUserID',
            'IP_addr' => 'InsertIPAddress',
            'Message' => array('Column' => 'Body'),
            'Format' => 'Format',
            'Message_date' => array('Column' => 'DateInserted')
        );
        $ex->exportTable('Comment', "
         select th.*
         from :_tblThread th", $comment_Map);
```

It's more efficient and also easier to read so simply do:

```
$ex->exportTable('Comment', "
    select 
        th.Thread_ID as CommentID,
        th.Topic_ID as CommentID,
        th.Thread_ID as DiscussionID,
        th.IP_addr as InsertIPAddress,
        th.Message as Body,
        th.Format as Format,
        th.Message_date as DateInserted,
    from :_tblThread th"
);
```
Basically we try to not use the "mapping" functionality unless we need to, when we use filters (`'Subject' => array('Column' => 'Name', 'Filter' => 'HTMLDecoder'),`) for example. You can probably apply this to most of your selects, Thanks for your help, let me know if you have any question !

**Comment by R-J** [comment]
Thanks for your hints, @topcheese13 and sorry for not looking at this earlier!

**Pull request merged**